TypeCobol.LanguageServer/TypeCobolServerHost.cs

```diff
@@ -296,6 +309,15 @@ static int Main(string[] args)
                 typeCobolServer.UseOutlineRefresh = UseOutlineRefresh;
                 typeCobolServer.UseCfgDfaDataRefresh = UseCfg;
 
+#if EUROINFO_RULES
+                try
+                { //Read Cpy Copy names file.
+                    typeCobolServer.CpyCopyNamesMap = TypeCobol.Tools.Options_Config.TypeCobolOptionSet.GetCpyCopiesFile(CPYMapFilePath);
+                } catch(Exception e)
+                {
+                    logWriter.WriteLine(e.Message);                    
+                }
+#endif
```

Comment:
No need to load early, see my other remark on TypeCobolServer

Reply:
done in e4234a0